packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -298,8 +300,36 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
         _getListTile(
           appLocalizations.account_delete,
           () async {
+            await showDialog<SmoothAlertDialog>(
+                context: context,
+                builder: (BuildContext context) {
+                  return SmoothAlertDialog(
+                    title: appLocalizations.account_delete,
+                    body: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: <Widget>[
+                        const Text(
+                            'Are you sure you want to delete your account? \n Is there a specific reason share below '),
```

Comment:
translations /

Reply:
yes sure !

Reply:
+ Please remove unused spaces (at the beginning / end of each sentences)